crates/host/src/wasmbus/mod.rs

```diff
@@ -1993,11 +1993,30 @@ impl Host {
                 }
             }
         });
+
+        // Process existing data without emitting events
+        data.keys()
```

Comment:
There is a race condition here and since it's probably better to process events at-least-once rather than at-most-once (we cannot do exactly-once I believe due to lack of "transactions" in NATS), this block should be placed *after* the `watch`

Reply:
Good point 👍